pyvista/report.py

```diff
@@ -0,0 +1,230 @@
+"""Module managing errors."""
+from collections import namedtuple
+import re
+import subprocess
+import sys
+import warnings
+
+import scooby
+from vtkmodules.vtkCommonCore import vtkVersion
+
+
+def VTKVersionInfo():
+    """Return the vtk version as a namedtuple."""
+    version_info = namedtuple('VTKVersionInfo', ['major', 'minor', 'micro'])
+
+    try:
+        ver = vtkVersion()
+        major = ver.GetVTKMajorVersion()
+        minor = ver.GetVTKMinorVersion()
+        micro = ver.GetVTKBuildVersion()
+    except AttributeError:  # pragma: no cover
+        warnings.warn("Unable to detect VTK version. Defaulting to v4.0.0")
+        major, minor, micro = (4, 0, 0)
+
+    return version_info(major, minor, micro)
+
+
+vtk_version_info = VTKVersionInfo()
```

Comment:
I'd never have thought to look in this module for `vtk_version_info`.

Reply:
Hm. I thought it was intuitive because this module is how/where we generate reports and get all the other versioning info for the system/other packages.

Perhaps this should go in `pyvista/core/_vtk_core.py`?

Reply:
It might just be me, but yeah, that would be a better place for me. But it's also possible that I'm the one who's weird here. In any case we can look at `type(vtk_version_info)` and that tells us where it's defined...

Reply:
Seems like this belongs here or in a utilities module (e.g. `core/utilities`).

PR is so big it's killing the web interface for me, recommend a follow-up PR if needed.